test/testsettings

```diff
@@ -0,0 +1 @@
+{"trackingEnaled":false}
```

Comment:
I don't think we should include this file to repo

Reply:
How would you propose to handle this in CI if we don't have this file?

Reply:
removed file and started generating from .sh